kybra/compiler/kybra_generate/src/py_ast/node/mod.rs

```diff
@@ -0,0 +1,43 @@
+use rustpython_parser::ast::{ExprKind, Located, StmtKind};
+
+use crate::source_map::SourceMapped;
+
+pub mod canister_method;
+pub mod data_type;
+pub mod external_canister;
+pub mod guard_function;
+pub mod param;
+pub mod stable_b_tree_map_nodes;
+
+pub use stable_b_tree_map_nodes::StableBTreeMapNode;
+
+impl SourceMapped<&Located<StmtKind>> {
```

Comment:
Should we implement this for the stmt kind instead of the sourcemapped located stmtkind?